### PR TITLE
NL-109 Connect Calendar to Backend

### LINKED ISSUE #102: Calendar Monthly Goals
## Name and Email
  - Name: Nicholas Vanny
  - Email: nvanny@ucsd.edu

## Complexity of Issue (Scale from 1-5)
  - 2
  
## Issue Description
  - Be able to edit and add month/year goals

### LINKED ISSUE #107: Connect CalendarJS to Backend
## Name and Email
  - Name: Rickesh Khilnani
  - Email: rickeshkhilnani@gmail.com

## Complexity of Issue (Scale from 1-5)
  - 2
  
## Issue Description
  -

### PR DESCRIPTION
**Pull request by Rick10101221**
Closes #107 #102

### CODE REVIEW

**Review by DustinLin — COMMENTED** [review]
We could move the `deprecated-source` directory entirely out of `source`, and have that somewhere else in the repo for record purposes

File: source/Backend/BackendInit.js [hunk 1]
```diff
@@ -308,6 +326,18 @@ async function getDataAtDBPath(path) {
     }
 }
 
+// TODO: Refactor getCurrentDay
```
Comment (DustinLin) [inline source/Backend/BackendInit.js:329]: what do you mean by this? Seems like these two functions will accomplish the same thing?

**Review by DustinLin — COMMENTED** [review]
I think we should name `General-JS` to be something related to custom HTML elements/HTML templates?

**Comment by DustinLin** [comment]
Looking at the live deployment site, I feel the highlighted day in the calendar is a bit too small, maybe make the border thicker?

**Comment by Rick10101221** [comment]
MUST MERGE #109

**Pull request merged**